css/style.css

```
@@ -129,65 +154,524 @@
+  margin-top: 39px;
+  margin-bottom: 73px;
+}
+
+.about {
+  display: grid;
+  grid-template-columns: 1fr min-content;
+  gap: 146px;
+}
+
+.about_left_column {
+  margin-top: 34px;
+}
+
+.about_slogan {
+  font-size: 45px;
+  line-height: 45px;
+  font-weight: 500;
+  color: var(--basic-black);
+  margin: 0 0 32px 0;
+}
+
+.about_orders {
+  font-weight: 700;
+  text-transform: uppercase;
+  margin: 0 0 23px;
+}
+
+.about_text {
+  margin: 0 0 40px;
+}
+
+.development {
+  list-style: none;
+  line-height: 48px;
+  margin: 0;
+  padding: 0;
+}
+
+.about_img {
+  margin-bottom: 37px;
+}
+
+.about_table {
+  font-weight: 700;
+  border-spacing: 0;
+}
+
+.about_table_caption {
+  color: var(--basic-black);
+  margin-bottom: 31px;
+}
+
+.about_table_percents {
+  font-size: 45px;
+  line-height: 64px;
+  color: var(--basic-black);
+}
+
+.about_table th {
+  text-align: left;
+  padding: 0 21px 11px 0;
+}
+
+.about_table th:last-of-type {
+  padding: 0 0 11px 0;
+}
+
+.about_table td {
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.about_table_percents sup {
+  font-size: 26px;
+  line-height: 40px;
+}
+
+/* Partners */
+
+.partners {
+  margin-top: 46px;
+  margin-bottom: 44px;
+}
+
+.partners_list {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  text-align: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.partners_item {
+  align-self: center;
+}
+
+.partners_item a {
+  color: var(--basic-black);
+  opacity: 0.2;
+}
+
+.partners_item a:hover {
+  color: var(--basic-black);
+  opacity: 1;
+}
+
+.partners_item a:active {
+  color: var(--basic-black);
+  opacity: 0.1;
+}
+
+/* Map */
+
+.conteiner_map {
+  width: 100%;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.map {
+  margin-top: 80px;
+  position: relative;
+}
+
+.map_map {
+  max-height: 416px;
+}
+
+.map_marker {
+  position: absolute;
+  top: 86px;
+  left: 822px;
+}
+
+/* Contacts */
+
+.contacts {
+  width: 319px;
+  height: 308px;
+  background-color: var(--basic-white);
+  position: absolute;
+  left: 140px;
+  top: 54px;
+  text-align: center;
+  overflow: auto;
+}
+
+.contacts_header {
+  font-size: 18px;
+  line-height: 30px;
+  font-weight: 700;
+  color: var(--basic-black);
+  text-transform: uppercase;
+  margin: 49px 0 23px 0;
+}
+
+.contacts_description {
+  line-height: 18px;
+  color: var(--special-lightgray);
+  font-style: normal;
+  margin-bottom: 23px;
+}
+
+.contacts_phone {
+  line-height: 18px;
+  color: var(--special-lightgray);
+  text-decoration: none;
+  margin: 0 0 37px 0;
+}
+
+.contacts_phone_wrapper {
+  margin: 0 0 37px 0;
+}
+
+.contacts_button {
+  display: flex;
+  justify-content: center;
+}
+
+/* Modal */
+
+.modal {
+  width: 960px;
+  height: 590px;
+  background-color: var(--basic-white);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--basic-black);
+  position: absolute;
+  top: 1706px;
+  left: 50%;
+  margin-left: -480px;
```

Так более универсально:
```suggestion
  transform: translateX(-50%)
```
Вообще может стоит позиционировать модальное окно фиксированно? Так пользователю будет удобнее, т.к. окно будет посреди экрана независимо от скролла страницы